### ISSUE TITLE
Optimize focus accessibility not working properly with Actionable Notification component

### ISSUE THREAD

**Issue by vitorwtavares** [camunda/camunda#32245]
### Component

Optimize

### Affected version

8.8, 8.7, 8.6, 8.5

### Severity

Low

### Likelihood

Low

### Description

This bug was found as part of a [support incident](https://jira.camunda.com/browse/SUPPORT-27242).

In the “Configure process” dialog box, it is not possible to navigate further from the close message button using the tab key. By pressing the TAB key, the focus remains on the same element and you can only navigate to the next interactive element after closing the message. Pressing the ESC key closes the entire dialog. This is not the expected behavior.

### Steps to reproduce

1. Navigate to Optimize's Dashboard.
2. Click on the configuration icon on the right side of any process on the list.
3. Try clicking the tab key until you reach the elements on the form.
4. Observe the focus does not reach the text input, but instead loops between the close modal button and the inline alert.

### Current behavior

User is unable to properly navigate between elements inside the configuration modal using only the tab key (accessibility issue), focus is drawn to invisible elements at times.

### Expected behavior

User is able to navigate between all elements inside the configuration using just the tab key and no invisible elements draw the focus.

### Environment

SaaS & SM

### Version

- Component Version: Affects all versions supported currrently (8.8, 8.7, 8.6, 8.5)

### Rootcause

This behavior is intentional due to the use of focus sentinels for "focus trapping" in the base component we use (Actionable Notification), that comes from Carbon Design System.

### Solution ideas

- Using a `Callout` variation of the [Notification component](https://react.carbondesignsystem.com/?path=/docs/components-notifications-actionable--overview) instead of the `Actionable` one.
- Keeping the current component and manually disabling the focus traps by changing the role of the component and disabling the focus sentinel elements via CSS to avoid invisible elements grabbing focus.

### Dev -> QA handover

- Resources:
- Versions to validate:
- Release version (in which version this feature will be released):

### Links

- [Support ticket](https://jira.camunda.com/browse/SUPPORT-27242)
